pkg/email/template/template.go

```diff
@@ -0,0 +1,34 @@
+package template
+
+import (
+	"strings"
+	"text/template"
+)
+
+const (
+	ResetPasswordPath  = "/auth/resetpassword?token="
+	ResetEmailTemplate = `Hello, <br>
+	A request has been received to change the password to your account,  <br>
+	Click on this link to reset your password <br>
+	{{.Host}}{{.ResetPasswordPath}}{{.Tkn}} <br>
+	If you did not initiate this request you can please diregard this email. <br>
+	Thanks  <br>
+	The LakeFS team <br>
+	`
+	ResetPasswordEmailSubject = "Reset Password Request for your Lakefs account"
+)
+
+type Link struct {
+	Host              string
+	ResetPasswordPath string
+	Tkn               string
+}
```

Comment:
Use `url.URL` to construct an url.
Base URL should be provided to access lakeFS - you can't resolve how the user is accessing the lakeFS.
Also, the 'url.URL' provides a way to set the query parameters to construct a valid URL.
Try to avoid URL building by string concatenation.